examples/crud-gorm/handlers/handler.go

```diff
@@ -0,0 +1,75 @@
+package handlers
+
+import (
+	"strconv"
+
+	"github.com/go-fuego/fuego"
+	"github.com/go-fuego/fuego/examples/crud-gorm/models"
+	"github.com/go-fuego/fuego/examples/crud-gorm/queries"
+)
+
+type Handlers struct {
+	UserQueries *queries.UserQueries
+}
+
+type UserToCreate struct {
+	Name  string `json:"name"`
+	Email string `json:"email"`
+}
+
+func (h *Handlers) GetUsers(c fuego.ContextNoBody) ([]models.User, error) {
+	return h.UserQueries.GetUsers()
+}
+
+func (h *Handlers) GetUserByID(c fuego.ContextNoBody) (models.User, error) {
+	id, err := strconv.Atoi(c.PathParam("id"))
+	if err != nil {
+		return models.User{}, fuego.BadRequestError{}
+	}
+	user, err := h.UserQueries.GetUserByID(uint(id))
+	if err != nil {
+		return models.User{}, err
+	}
+	return *user, nil
```

Comment:
this could be, if you change your return type.

```suggestion
	return h.UserQueries.GetUserByID(uint(id))
```

but I'm not familiar with gorm

Reply:
Added myself on the PR. 

FYI I don't know GORM myself, I used strategy detailed here https://stackoverflow.com/questions/53684573/how-to-return-the-record-that-i-create-it-with-create-of-gorm